ur10_moveit_config/config/ompl_planning.yaml

```diff
@@ -34,7 +34,6 @@ manipulator:
     - TRRTkConfigDefault
     - PRMkConfigDefault
     - PRMstarkConfigDefault
-  projection_evaluator: joints(shoulder_pan_joint,shoulder_lift_joint)
```

Comment:
This was the suggestion of the OMPL developers. I guess that removing the projection makes the robot "fall" into the most generic class, so I hope that if they change the default algorithm for that class, it would still behave best for the robot among the available planning algorithms.

But I am actively looking for a way to be more specific.